openscenario/openscenario_interpreter/src/syntax/maneuver.cpp

```diff
@@ -29,6 +30,19 @@ Maneuver::Maneuver(const pugi::xml_node & node, Scope & scope)
   });
 }
 
+auto Maneuver::run() -> void
+{
+  for (size_t index{0}; auto && event : elements) {
```

Comment:
This repository is compiled with C++17. Don't take advantage of C++20 features yet.
```suggestion
    std::size_t index = 0;
    for (auto && event : elements) {
```